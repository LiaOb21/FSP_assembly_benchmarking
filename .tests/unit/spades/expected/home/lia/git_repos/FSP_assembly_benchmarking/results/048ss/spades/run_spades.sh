set -e
true
true
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/spades-core /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K21/configs/config.info
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/spades-core /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K33/configs/config.info
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/spades-core /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/configs/config.info
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3 /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/copy_files.py /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/before_rr.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/before_rr.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/assembly_graph_after_simplification.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/assembly_graph_after_simplification.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/final_contigs.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/contigs.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/first_pe_contigs.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/first_pe_contigs.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/strain_graph.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/strain_graph.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/scaffolds.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/scaffolds.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/scaffolds.paths /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/scaffolds.paths /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/assembly_graph_with_scaffolds.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/assembly_graph_with_scaffolds.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/assembly_graph.fastg /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/assembly_graph.fastg /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/final_contigs.paths /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/contigs.paths
true
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3 /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/breaking_scaffolds_script.py --result_scaffolds_filename /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/scaffolds.fasta --misc_dir /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/misc --threshold_for_breaking_scaffolds 3
true
