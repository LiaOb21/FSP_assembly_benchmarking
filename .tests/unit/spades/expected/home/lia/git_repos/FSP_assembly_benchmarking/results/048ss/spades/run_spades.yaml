- STAGE: Before start
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Before_start_LC2RMUNPYH8SNN217089JHQEXHXV9ST4
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: before_start
- STAGE: Assembling
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Assembling_A97NSEU038G4TA9HJPX2R1O6E315NI59
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: as_start
- STAGE: K21
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K21/configs/config.info
  config_dir: K21
  del_after: []
  job_uuid: hpcSPAdes_K21_SWBQGETDEIRFSWIQI7Q4WMIAJMRUIG6J
  mpi_support: true
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/{spades_core}
  short_name: k21
- STAGE: K33
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K33/configs/config.info
  config_dir: K33
  del_after: []
  job_uuid: hpcSPAdes_K33_OYC3PY25N09STBCOIQ9BP21Y0BK8NCB3
  mpi_support: true
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/{spades_core}
  short_name: k33
- STAGE: K55
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/configs/config.info
  config_dir: K55
  del_after: []
  job_uuid: hpcSPAdes_K55_V7XHG7AHMPLT8JMB38XD83GGQFESCKZ6
  mpi_support: true
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/{spades_core}
  short_name: k55
- STAGE: Copy files
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/copy_files.py
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/before_rr.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/before_rr.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/assembly_graph_after_simplification.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/assembly_graph_after_simplification.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/final_contigs.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/contigs.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/first_pe_contigs.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/first_pe_contigs.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/strain_graph.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/strain_graph.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/scaffolds.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/scaffolds.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/scaffolds.paths
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/scaffolds.paths
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/assembly_graph_with_scaffolds.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/assembly_graph_with_scaffolds.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/assembly_graph.fastg
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/assembly_graph.fastg
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/K55/final_contigs.paths
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/contigs.paths
  config_dir: ''
  del_after:
  - .bin_reads
  - tmp/spades_un35vgri
  job_uuid: SPAdes_Copy_files_HFLOHPHKAV3XGVHQLWZC2HNH1FW9C290
  mpi_support: false
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3
  short_name: copy_files
- STAGE: Assembling
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Assembling_FN28R3NNWF9A59NK1KKJ1L29FQ5DHN45
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: as_finish
- STAGE: Breaking scaffolds
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/breaking_scaffolds_script.py
  - --result_scaffolds_filename
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/scaffolds.fasta
  - --misc_dir
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ss/spades/misc
  - --threshold_for_breaking_scaffolds
  - '3'
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Breaking_scaffolds_EEJPJFDXM3AW4R1LGEFCH1W3JSRT9QCI
  mpi_support: false
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3
  short_name: bs
- STAGE: Terminate
  args: []
  config_dir: ''
  del_after:
  - configs
  job_uuid: SPAdes_Terminate_0YGH0KDIV1QLWYK1C3522SL9LAB0J0BV
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: terminate
